doc/themes/the-things-stack/assets/css/theme.scss

```diff
@@ -367,7 +367,7 @@ details + blockquote {
   padding: 12px;
 }
 
-.new-in-version, .distribution-inline {
+.new-in-version, .distribution-inline, .deprecated {
```

Comment:
This needs to be styled differently, to communicate caution. @pierrephz can you propose a simple CSS class for such warnings?

Reply:
![deprecation-warning](https://user-images.githubusercontent.com/65215579/99782145-10257880-2b19-11eb-92a7-400aadbe384f.png)
How about we keep the format and just change the color to something like this?

Reply:
this is too much here, lets be more subtle and use a red for caution.
here is what I suggest:

![image](https://user-images.githubusercontent.com/26456318/99807316-a53b6800-2b3f-11eb-83a1-f45c1193aada.png)

don't use ITC Avant Garde here, just play with font-weight

cc @nejraselimovic

Reply:
> this is too much here, lets be more subtle and use a red for caution.
> here is what I suggest:
> 
> ![image](https://user-images.githubusercontent.com/26456318/99807316-a53b6800-2b3f-11eb-83a1-f45c1193aada.png)
> 
> don't use ITC Avant Garde here, just play with font-weight
> 
> cc @nejraselimovic

Well, that indeed looks more subtle :D can you please push a commit with that one?